SU-SWS/stanford_slides
# HSDO-13: Removed duplicate old manage slides view

# READY
- Removes the old view.
- Adds update hook to save the view to the db for the old users.

**To test:**
- Enable Install with the 7.x-1.x branch
- Change to this branch
- Run updates
- See view get saved to DB
